doc/source/create-k8s-cluster.rst

```diff
@@ -163,6 +169,121 @@ Setting up Kubernetes on Microsoft Azure Container Service (ACS)
 
    The response should list three running nodes.
 
+.. _amazon-aws:
+
+Setting up Kubernetes on Amazon Web Services (AWS)
+--------------------------------------------------
+
+AWS does not have native support for Kubernetes, however there are
+many organizations that have put together their own solutions and
+guides for setting up Kubernetes on AWS. We like the `Heptio guide`_,
+and recommend using this for setting up your stack.
+
+.. note::
+
+  See the `Heptio's note on security <http://blog.kubernetes.io/2016/08/security-best-practices-kubernetes-deployment.html>`_
```

Comment:
The name of this link is confusing, because the link takes you to a page that has nothing to do with heptio (as far as I could tell).